platform/integrations/rancher/configuration.mdx

```diff
@@ -110,7 +110,7 @@ The Rancher integration can be enabled on a per-project basis. Subsequently, any
     Click on <NavStep>Rancher</NavStep> and toggle the <Label>Enable Rancher Integration</Label> switch to enable synchronization for this project. Additional configuration options will appear.
   </Step>
   <Step>
-    Add the <Label>Rancher Project ID</Label> and <Label>Rancher Cluster ID</Label> of the Rancher project you want to connect.
+    Add the <Label>Rancher Project ID</Label> and <Label>Rancher Cluster ID</Label> of the Rancher project you want to connect. Note that those are different from the project name and cluster name. To retrieve those, you can either extract them from the URL in the Rancher dashboard, or using the rancher CLI.
```

Comment:
Do we have explicit steps on how to get this data?

For example, where in the URL is it? Or what Rancher CLI command is it to get that info?